README.md

```diff
@@ -0,0 +1,188 @@
+
+# Sample Android Native Application
+
+## Introduction
+
+This is a sample Android Java application showing use of DataSync, [Keycloak](https://www.keycloak.org/about.html) and Unified Push using native upstream SDK's.
+Application is sending requests to [Ionic showcase server]([https://github.com/aerogear/ionic-showcase/tree/master/server](https://github.com/aerogear/ionic-showcase/tree/master/server)) which is a GraphQL server. For DataSync, application uses Apollo Client to query, mutate and subscribe. [AppAuth](https://github.com/openid/AppAuth-Android) to connect
+ with Keycloak and Aerogear SDK for Unified Push support.
```

Comment:
I think using a bullet list here to list each of the service and upstream SDK used will be better as it will make it easier to read.

Reply:
good idea, thank Wei